<!DOCTYPE html>
<html lang="en-US">
<head>
<title>html from pdf</title>
<meta charset="utf-8">
<style>body{font-size: 18px;}</style>
<body>
<div data-page='1' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 1</p><p>R E S E A R C H Open Access
</p>
<p>Attenuation of multiple Nef functions in HIV-1
elite controllers
</p>
<p>Philip Mwimanzi1,2†, Tristan J Markle2†, Eric Martin2, Yoko Ogata1, Xiaomei T Kuang2, Michiyo Tokunaga1,
Macdonald Mahiti1, Florencia Pereyra3, Toshiyuki Miura4, Bruce D Walker3, Zabrina L Brumme2,5,
</p>
<p>Mark A Brockman2,5*and Takamasa Ueno1*
</p>
<p>Abstract
</p>
<p>Background:Impaired HIV-1 Gag, Pol, and Env function has been described in elite controllers (EC) who
spontaneously suppress plasma viremia to &lt; 50 RNA copies/mL; however, activity of the accessory protein Nef
remains incompletely characterized. We examined the ability of 91 Nef clones, isolated from plasma of 45 EC and
46 chronic progressors (CP), to down-regulate HLA class I and CD4, up-regulate HLA class II invariant chain (CD74),
enhance viral infectivity, and stimulate viral replication in PBMC.
</p>
<p>Results:In general, EC Nef clones were functional; however, all five activities were significantly lower in EC
compared to CP. Nef clones from HLA-B*57-expressing EC exhibited poorer CD4 down-regulation function
compared to those from non-B*57 EC, and the number of EC-specific B*57-associated Nef polymorphisms
correlated inversely with 4 of 5 Nef functions in these individuals.
</p>
<p>Conclusion:Results indicate that decreased HIV-1 Nef function, due in part to host immune selection pressures,
may be a hallmark of the EC phenotype.
</p>
<p>Keywords:HIV-1, Nef, Elite controllers, Human Leukocyte Antigen (HLA) class I, Immune escape, Replication
capacity, HLA-B*57
</p>
<p>Background
</p>
<p>Elite controllers (EC) are rare (&lt;1%) HIV-1 infected
individuals who spontaneously suppress plasma viral loads
to undetectable levels in the absence of antiviral therapy.
Several factors likely contribute to this phenotype, including
host genetics [1], characteristics of HLA-restricted T-cell
responses [2], immune-mediated reductions in viral protein
function and/or replication [3,4], and acquisition of attenu-
ated viruses [5,6]. Recombinant viruses expressinggag and
polsequences from EC exhibit reducedin vitroreplication
capacity, due in part to cytotoxic T lymphocyte (CTL)
escape mutations selected by certain HLA class I (HLA-I)
alleles [3,4], while EC-derived viral envelopes exhibit
impaired entry [7]. The in vitro function of other viral
proteins in EC remains incompletely characterized.
</p>
<p>HIV-1 Nef is an accessory protein required for main-
tenance of high viral loads and progression to AIDS [8],
as demonstrated by slow or non-progressive disease in
hosts infected withnef-deleted or otherwisenef-defective
strains [5,6,9,10]. Nef exhibits a variety ofin vitro func-
tions that may modulate pathogenesis, including CD4
down-regulation [11], HLA-I down-regulation [12], HLA
class II invariant chain (CD74) up-regulation [13],
enhancement of virion infectivity [14], and stimulation
of viral replication in PBMC [15] (for reviews see
[16-18]). Multiple Nef activities may act together to
facilitate immune evasion and enhancement of viral
spread in vivo [19]; however, multi-functional assess-
ments of patient-derived Nef clones from HIV elite
controllers are lacking. Although Nef sequence diversity
is highly influenced by host HLA-I selection pressures
[20], the relationship between HLA-associated poly-
morphisms and Nef function is largely unknown. Asses-
sing multiple in vitroNef functions in EC, a population
that is highly enriched for protective HLA-I alleles such
</p>
<p>* Correspondence:mark_brockman@sfu.ca;uenotaka@kumamoto-u.ac.jp
</p>
<p>†<sub>Equal contributors</sub>
</p>
<p>2<sub>Simon Fraser University, Burnaby, BC, Canada</sub>
1
</p>
<p>Center for AIDS Research, Kumamoto University, 2-2-1 Honjo, Kumamoto
860-0811, Japan
</p>
<p>Full list of author information is available at the end of the article
</p>
<p>© 2013 Mwimanzi et al.; licensee BioMed Central Ltd. This is an Open Access article distributed under the terms of the
Creative Commons Attribution License (http://creativecommons.org/licenses/by/2.0), which permits unrestricted use,
distribution, and reproduction in any medium, provided the original work is properly cited.
</p>
</div>
<div data-page='2' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 2</p><p>as B*57 [1], provides an opportunity to investigate these
questions.
</p>
<p>Previous analysis of plasma HIV RNA Nef sequences in
our cohort of EC revealed no evidence of gross mutational
defects [21], suggesting that any impairment in Nef protein
function would have a more complex etiology. For this
study, we generated recombinant viruses encoding a single
representative HIV RNA Nef clone from 45 EC to assess
Nef-mediated down-regulation of HLA class I, up-
regulation of HLA class II invariant chain (CD74), viral in-
fectivity, and viral replication in PBMC. The same Nef
clone was engineered into a GFP-expression vector to
assess its ability to down-regulate CD4. Results were com-
pared to the activities of HIV RNA-derived Nef clones from
46 chronic progressors (CP). Finally, we assessed the role of
host immune selection pressures, most notably novel poly-
morphisms associated with HLA-B*57 in EC, on Nef func-
tion in these individuals.
</p>
<p>Results
</p>
<p>Nef protein expression and viral production
</p>
<p>For each of 45 EC and 46 CP, a single representative plasma
HIV RNA-derived Nef sequence with an intact open
</p>
<p>reading frame (ORF) was cloned into a recombinant NL4.3
virus construct. Consistent with previous analyses of bulk
plasma HIV RNA sequences from our EC cohort [21],
clonal Nef sequences from EC showed no evidence of gross
defects or recent shared ancestry (Figure 1, Additional
file 1: Table S1). Western blots revealed comparable band
intensities between EC and CP, indicating that EC Nefs
were not markedly diminished in steady-state protein
expression levels (Figure 2A, B). Similarly, p24Gaglevels in
culture supernatants were comparable between groups,
indicating that EC Nefs were not significantly impaired in
virion production (Figure 2C).
</p>
<p>Nef-mediated enhancement of viral infectivity and
replication is impaired in EC
</p>
<p>All viruses harboring EC Nef displayed infectivity greater
than the negative control NL4.3ΔNef, which had 7.3% in-
fectivity relative to NL4.3-NefSF2. Compared to control
</p>
<p>strain NL4.3-NefSF2, median EC Nef infectivity was 55%
</p>
<p>(IQR 38-76%), values that were significantly lower than CP-
derived Nef (median 116, IQR 88-160%) (p &lt; 0.001;
Figure 3A). Similarly, all viruses harboring EC Nef displayed
higher replication capacities than the negative control
</p>
<p>Figure 1Maximum-likelihood phylogenetic tree of plasma HIV RNA-derived Nef clonal sequences.EC-derived Nefs are red, CP-derived
Nefs are blue, and control strain SF2 is black.
</p>
</div>
<div data-page='3' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 3</p><p>NL4.3ΔNef in PBMCs from four HIV-negative donors.
Consistent with previous reports [22,23], replication of
patient-derived Nef recombinant viruses in PBMC differed
to some extent among donors; however, viruses encoding
EC Nef displayed consistently poorer ability to replicate in
PBMC relative to those harboring CP Nef in all donors
(p≤0.01; Figure 3B, C). Averaged over all four donors, me-
dian [IQR] replication capacities were 34 [23–52]% of
NL4.3-NefSF2 for EC-derived viruses and 76 [57–98]% for
</p>
<p>CP-derived viruses, respectively (p &lt; 0.001, not shown).
</p>
<p>Modulation of surface HLA-I, CD74, and CD4 by EC Nef
All EC Nef clones displayed greater ability to modulate
cell-surface receptors thanΔNef negative controls. Relative
to control strain NL4.3-NefSF2, EC-derived Nef recombinant
</p>
<p>viruses maintained considerable HLA-I down-regulation
activity (median 95 [IQR 79–106]%) that was nevertheless
significantly lower compared to CP Nef viruses (median
106 [IQR 96–111]%) (p&lt;0.001; Figure 4A, B). The ability of
EC Nef viruses to up-regulate CD74 was markedly lower
(median 49 [IQR 35–76]%) compared to CP Nef viruses
(median 111 [IQR 68–150]%) (p &lt; 0.001; Figure 4C, D).
HIV-1 Vpu and Env proteins contribute to surface CD4
modulation [24]; therefore, Nef-mediated CD4 down-
regulation activity was assessed using DNA expression plas-
mids. Relative to control NefSF2, most EC Nef clones main-
</p>
<p>tained substantial CD4 down-regulation activity (median 91
[IQR 76–95]%) that was nevertheless significantly lower
compared to CP Nef clones (median 99 [IQR 89–101]%)
(p = 0.002) (Figure 4E, F). All EC Nef sequences and func-
tional data are provided in Additional file 1: Table S1. Of
interest, 32 of 45 (71%) EC Nef viruses displayed replication
activity less than 50% of that of control strain NefSF2, while
</p>
<p>only one EC Nef virus showed HLA-I down-regulation
activity less than 50%. Three EC Nef demonstrated activity
less than 50% of NefSF2 for all functions tested, except
</p>
<p>HLA-I down-regulation (Additional file 1: Table S1).
</p>
<p>Host HLA-I allele expression and Nef function in EC
Protective HLA-I alleles, most notably B*57, are over-
represented in EC [1,25]. To investigate this as a potential
confounder in comparisons between EC and CP, we re-
analyzed our data excluding individuals who expressed
HLA-B*57 (17/45 of EC and 8/46 of CP). Measures for all
five Nef functions remained significantly lower among non-
B*57 EC compared to non-B*57 CP (all p &lt; 0.01, not
shown). Exclusion of individuals expressing any protective
allele (defined as B*27, B*57, and B*58:01) yielded similar
results (all p &lt; 0.05, not shown).
</p>
<p>Immune selection by protective HLA-I alleles, including
B*57, can modulate thein vitrofunction of certain HIV-1
proteins in EC [3,4]. To examine whether this was also true
for Nef, we stratified EC Nef clones by host B*57 expression
and observed significantly lower CD4 down-regulation
activity in B*57-derived compared to non-B*57-derived EC
Nefs (median [IQR] 83 [55–94]% for B*57 vs. 92 [83–97]%
for non-B*57 EC, respectively, p = 0.038). Significant differ-
ences were not seen for the other Nef activities tested
(Figure 5). Of 20 HLA-I alleles expressed in a minimum of
five EC, correlations with Nef function were also observed
for C*06 (in linkage disequilibrium with B*57; median 74 vs
93% CD4 down-regulation activity in C*06 vs. non-C*06
EC) and A*01 (median 83 vs 97% HLA-I down-regulation
activity in A*01 vs. non-A*01 EC) (both p &lt; 0.05; q &lt; 0.05).
No HLA-I associations were observed for Nef-mediated in-
fectivity, replication, or CD74 up-regulation activity in EC.
</p>
<p>Unique HLA-associated polymorphisms and Nef function
in EC
</p>
<p>Modulation of viral protein function in EC by protective
HLA-I alleles may be due to the selection of unconven-
tional HLA-associated polymorphisms in this patient group
[26,27]. To examine this, we applied phylogenetically-
corrected methods [28] to identify HLA-B*57-associated
Nef polymorphisms in our cohort of 45 EC. Nine
</p>
<p><b>EC Nefs</b>
<b>Sheep </b>
</p>
<p>α-Nef
</p>
<p><b>Rabbit </b>
α-Nef
</p>
<p><b>CP Nefs</b>
Δ Nef
</p>
<p><b>Nef-SF2</b>
</p>
<p><b>A</b>
</p>
<p><b>EC</b> <b>CP</b> <b>EC</b> <b>CP</b>
<b>0</b>
</p>
<p><b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
<b>120</b>
<b>140</b>
</p>
<p><b>R</b>
</p>
<p><b>e</b>
</p>
<p><b>la</b>
</p>
<p><b>tiv</b>
</p>
<p><b>e</b>
</p>
<p><b> ba</b>
</p>
<p><b>nd int</b>
</p>
<p><b>e</b>
</p>
<p><b>ns</b>
</p>
<p><b>it</b>
</p>
<p><b>y</b>
</p>
<p><b>EC</b> <b>CP</b>
</p>
<p><b>0</b>
<b>1000</b>
<b>2000</b>
<b>3000</b>
<b>4000</b>
<b>5000</b>
</p>
<p><b>V</b>
</p>
<p><b>ir</b>
</p>
<p><b>us</b>
</p>
<p><b> pr</b>
</p>
<p><b>oduc</b>
</p>
<p><b>ti</b>
</p>
<p><b>on (</b>
</p>
<p><b>p2</b>
</p>
<p><b>4</b>
</p>
<p><b> ng/m</b>
</p>
<p><b>l)</b>
</p>
<p><b>B</b> <b>C</b>
</p>
<p><b>Sheep </b>
</p>
<p>α-Nef <b>Rabbit α-Nef</b>
<b>p= 0.4</b>
</p>
<p><b>p= 0.2</b>
<b>p= 0.3</b>
</p>
<p>Figure 2Nef expression and progeny virus production.PanelA: Western blot detection of control strains NL4.3-NefSF2and NL4.3-ΔNef and
</p>
<p>two representative EC and CP-derived Nefs using two different polyclonal anti-Nef sera.PanelB: Relative band intensity of EC (red) and CP (blue)-
derived Nefs as detected by Western blot. Open circles identify the representative data shown in panel A. Bars depict median and interquartile
ranges. Statistical significance assessed using Mann–Whitney U-Test.PanelC: Virus production (measured as p24Gagin culture supernatant) of EC
(red) and CP (blue) Nef recombinant viruses.
</p>
</div>
<div data-page='4' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 4</p><p>associations were observed at p &lt; 0.05 (q &lt; 0.4) in B*57+
EC (Figure 6A). With the exception of V85L, these
B*57-associated polymorphisms were distinct from
those previously identified in large population-level
analyses of chronically subtype B infected individuals
(N &gt;1500) [20,29], suggesting that they may be largely
unique to EC. In contrast, a search for B*57-associated
polymorphisms in our cohort of 46 CP revealed several
expected Nef polymorphisms at p &lt; 0.05, including
V85L and H116N [20,29,30] (not shown), supporting
our ability to identify HLA-associated polymorphisms
in cohorts of the present size. Therefore, we reasoned
that the unconventional B*57-associated polymorphisms
observed in EC merited further attention.
</p>
<p>Among B*57-expressing EC (N = 17), we observed sig-
nificant inverse relationships between the number of EC-
specific B*57-associated polymorphisms and Nef-mediated
replication (Spearman R = −0.5, p = 0.04), HLA-I down-
regulation (R = −0.57, p = 0.02) and CD74 up-regulation
(R =−0.6, p = 0.01); and a modest, albeit not statistically
significant, negative correlation with CD4 down-regulation
(R =−0.37, p = 0.1) (Figure 6). No similar correlations were
observed between EC-specific polymorphisms associated
with other HLA-I alleles observed at comparable frequen-
cies in our cohort (i.e. A*01, A*02, A*03, A*30, C*06, C*07)
and Nef function in EC expressing these alleles (not
shown).
</p>
<p>Amino acids associated with EC Nef function
</p>
<p>To investigate the relationship between Nef sequence
and function in EC, we performed an exploratory ana-
lysis to correlate amino acids with each of the five func-
tions tested, regardless of host HLA. A total of 23
polymorphisms occurring at 14 sites were associated
with Nef-mediated modulation of HLA-I, CD74 and
CD4 (p &lt; 0.05, q &lt; 0.4; Table 1). No Nef polymorphisms
associated with infectivity or replication were observed
at this threshold.
</p>
<p>Discussion
</p>
<p>We assessed five in vitro Nef functions using clonal
plasma HIV RNA sequences from 45 EC and 46 CP. We
observed that EC Nef clones were generally functional,
especially for Nef’s most characteristic activities, CD4
and HLA-I down-regulation. Nevertheless, median EC
Nef activities were significantly lower for all five func-
tions when compared to those from CP. Median CP Nef
activities were consistent with that of HIV-1 strain SF2
used as a normalization control for all assays, indicating
that our selection of chronic Nef clones is representative
of chronic Nef isolates examined previously. The range
in Nef activities observed here may help to resolve dis-
crepancies between previous studies of HIV long-term
non-progressors or controllers, which have reported
</p>
<p><b>EC</b> <b>CP</b>
</p>
<p><b>0</b>
<b>100</b>
<b>200</b>
<b>300</b>
</p>
<p><b>R</b>
</p>
<p><b>e</b>
</p>
<p><b>la</b>
</p>
<p><b>ti</b>
</p>
<p><b>v</b>
</p>
<p><b>e</b>
</p>
<p><b> inf</b>
</p>
<p><b>e</b>
</p>
<p><b>c</b>
</p>
<p><b>tiv</b>
</p>
<p><b>it</b>
</p>
<p><b>y</b>
</p>
<p><b> t</b>
</p>
<p><b>o</b>
</p>
<p><b> c</b>
</p>
<p><b>ont</b>
</p>
<p><b>rol (</b>
</p>
<p><b>%</b>
</p>
<p><b>)</b>
</p>
<p><b>A</b>
</p>
<p><b>p&lt; 0.001</b>
</p>
<p><b>6</b> <b>9</b> <b>12</b>
</p>
<p><b>0</b>
<b>50</b>
<b>100</b>
<b>150</b>
<b>200</b>
<b>250</b>
<b>300</b>
<b>350</b>
</p>
<p><b>p</b>
</p>
<p><b>24 (</b>
</p>
<p><b>n</b>
</p>
<p><b>g</b>
</p>
<p><b>/ml</b>
</p>
<p><b>)</b>
</p>
<p><b>EC CP EC CP EC CP EC CP</b>
<b>0</b>
</p>
<p><b>50</b>
<b>100</b>
<b>150</b>
<b>200</b>
</p>
<p><b>250</b> <b>p&lt; 0.001</b>
</p>
<p><b>p&lt; 0.001</b>
</p>
<p><b>p= 0.01</b>
</p>
<p><b>p&lt; 0.001</b>
</p>
<p><b>D</b>
</p>
<p><b>onor</b>
</p>
<p><b> 1</b>
</p>
<p><b>D</b>
</p>
<p><b>onor</b>
</p>
<p><b> 2</b>
</p>
<p><b>D</b>
</p>
<p><b>onor</b>
</p>
<p><b> 3</b>
</p>
<p><b>D</b>
</p>
<p><b>onor</b>
</p>
<p><b> 4</b>
</p>
<p><b>R</b>
</p>
<p><b>e</b>
</p>
<p><b>la</b>
</p>
<p><b>ti</b>
</p>
<p><b>ve r</b>
</p>
<p><b>e</b>
</p>
<p><b>p</b>
</p>
<p><b>li</b>
</p>
<p><b>c</b>
</p>
<p><b>at</b>
</p>
<p><b>io</b>
</p>
<p><b>n</b>
</p>
<p><b>c</b>
</p>
<p><b>a</b>
</p>
<p><b>p</b>
</p>
<p><b>a</b>
</p>
<p><b>c</b>
</p>
<p><b>ity</b>
</p>
<p><b> to</b>
</p>
<p><b> c</b>
</p>
<p><b>o</b>
</p>
<p><b>n</b>
</p>
<p><b>tr</b>
</p>
<p><b>o</b>
</p>
<p><b>l (%</b>
</p>
<p><b>)</b>
</p>
<p><b>B</b>
</p>
<p><b>C</b>
</p>
<p>Δ <b>Nef</b>
<b>Nef-SF2</b>
</p>
<p><b>Days post infection</b>
</p>
<p>Figure 3Ability of EC Nef to enhance viral infectivity and
stimulate viral replication in PBMC.PanelA: Scatterplots depict
the ability of EC (red) and CP (blue) Nef recombinant viruses to
enhance viral infectivity. Values are normalized to that of NL4.3-
NefSF2such that values of 100% indicate infectivity equal to that of
</p>
<p>NL4.3-NefSF2, while values &lt;100% and &gt;100% indicate infectivity
</p>
<p>lower than or higher than that of NL4.3-NefSF2, respectively. Bars
</p>
<p>represent median and interquartile ranges. The p-value was
calculated using Mann–Whitney U-test.PanelB: Growth curves are
shown for EC (red) and CP (blue) Nef recombinant viruses, plus
control viruses NL4.3-NefSF2and NL4.3-ΔNef, in PBMCs from donor
</p>
<p>#1. The mean value from quadruplicate samples is shown at each
time point. The means ± SEM values for control viruses NL4.3-NefSF2
</p>
<p>and NL4.3-ΔNef at day 9 and 12 are 115 ± 27 and 1.3 ± 0.02, and
285 ± 18.8 and 4.3 ± 0.4 respectively.PanelC: Scatterplots depict
the ability of recombinant EC (red) and CP (blue) Nef recombinant
viruses to stimulate viral replication in PBMC from four HIV-negative
donors. Values are normalized to that of control NL4.3-NefSF2. Bars
</p>
<p>represent median and interquartile ranges. P-values were calculated
using Mann–Whitney U-test.
</p>
</div>
<div data-page='5' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 5</p><p>relative preservation of CD4 and/or HLA-down-
regulation function [32,33], inefficient Nef-mediated
CD4 and/or HLA-down-regulation [34-36] and reduced
infectivity [35] compared to CP. Our data suggest that
there is in vivopressure on Nef in EC to maintain CD4
and HLA-I down-regulation functions.
</p>
<p>Relative functional impairments between EC and CP
clones are not likely to be due to differences in Nef pro-
tein stability or expression levels, nor to recent descent
from a defective common ancestor. Similarly, while en-
richment of protective HLA alleles in EC may contribute
to Nef sequence, it is not likely to be the only explan-
ation for relative functional attenuation observed here,
since differences between groups persisted after persons
who expressed protective HLA alleles were excluded
</p>
<p>from analysis. Indeed, although significantly lower CD4
down-regulation activity was observed in B*57 compared
to non-B*57 EC (Figure 5E), this was not true for other
Nef functions, indicating that B*57 expression alone
does not guarantee Nef attenuation in this group.
</p>
<p>Rather, our results are consistent with functional vari-
ability of naturally occurring Nef sequences from EC,
which may be attributable in part to non-canonical
HLA-associated escape mutations selected in this rare
group. Previously, in order to investigate the influence of
HLA-associated viral polymorphisms on HIV-1 protein
function in EC, we have made use of reference lists of
common HLA-associated polymorphisms derived from
population-level studies of chronically infected indivi-
duals [20,29]. However, such lists may not capture rare
</p>
<p><b>EC</b> <b>CP</b>
</p>
<p><b>0</b>
<b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
<b>120</b>
<b>140</b>
</p>
<p><b>R</b>
</p>
<p><b>e</b>
</p>
<p><b>la</b>
</p>
<p><b>ti</b>
</p>
<p><b>ve act</b>
</p>
<p><b>ivi</b>
</p>
<p><b>ty t</b>
</p>
<p><b>o</b>
</p>
<p><b> co</b>
</p>
<p><b>n</b>
</p>
<p><b>tr</b>
</p>
<p><b>o</b>
</p>
<p><b>l (</b>
</p>
<p><b>%</b>
</p>
<p><b>)</b>
</p>
<p><b>EC</b> <b>CP</b>
</p>
<p><b>0</b>
<b>50</b>
<b>100</b>
<b>150</b>
<b>200</b>
<b>250</b>
<b>300</b>
<b>350</b>
</p>
<p><b>R</b>
</p>
<p><b>el</b>
</p>
<p><b>a</b>
</p>
<p><b>ti</b>
</p>
<p><b>ve act</b>
</p>
<p><b>ivi</b>
</p>
<p><b>ty t</b>
</p>
<p><b>o</b>
</p>
<p><b> co</b>
</p>
<p><b>n</b>
</p>
<p><b>tr</b>
</p>
<p><b>o</b>
</p>
<p><b>l (</b>
</p>
<p><b>%</b>
</p>
<p><b>)</b>
</p>
<p><b>EC</b> <b>CP</b>
</p>
<p><b>0</b>
<b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
<b>120</b>
</p>
<p><b>R</b>
</p>
<p><b>el</b>
</p>
<p><b>a</b>
</p>
<p><b>ti</b>
</p>
<p><b>ve act</b>
</p>
<p><b>ivi</b>
</p>
<p><b>ty t</b>
</p>
<p><b>o</b>
</p>
<p><b> co</b>
</p>
<p><b>n</b>
</p>
<p><b>tr</b>
</p>
<p><b>o</b>
</p>
<p><b>l (</b>
</p>
<p><b>%</b>
</p>
<p><b>)</b>
</p>
<p><b>Uninfected</b> Δ<b>Nef</b> <b>Nef-SF2</b>
</p>
<p><b>No DNA</b> <sub>Δ</sub><b>Nef</b> <b>Nef-SF2</b>
</p>
<p><b>B</b>
</p>
<p><b>D</b>
</p>
<p><b>F</b>
</p>
<p><b>HL</b>
</p>
<p><b>A-</b>
</p>
<p><b>I</b>
</p>
<p><b>CD7</b>
</p>
<p><b>4</b>
</p>
<p><b>CD4</b>
</p>
<p><b>HLA-I down-regulation</b>
</p>
<p><b>CD74 up-regulation</b>
</p>
<p><b>CD4 down-regulation</b>
<b>p&lt; 0.001</b>
</p>
<p><b>p&lt; 0.001</b>
</p>
<p><b>p&lt; 0.001</b>
</p>
<p><b>Uninfected</b> Δ<b>Nef</b> <b>Nef-SF2</b>
</p>
<p><b>0% </b>
<b>activity</b>
</p>
<p><b>100% </b>
<b>activity</b>
</p>
<p><b>0% </b>
<b>activity</b>
</p>
<p><b>100% </b>
<b>activity</b>
</p>
<p><b>0% </b>
<b>activity</b>
</p>
<p><b>100% </b>
<b>activity</b>
</p>
<p><b>p24</b>
</p>
<p><b>p24</b>
</p>
<p><b>GFP</b>
</p>
<p><b>A</b>
</p>
<p><b>C</b>
</p>
<p><b>E</b>
</p>
<p>Figure 4Ability of EC Nef to modulate cell surface receptor levels.PanelA: Flow cytometry plots depicting representative staining of cell-
surface HLA-I (HLA-A*2402; y-axis) vs. intracellular p24Gag<sub>(x-axis) for uninfected, NL4.3-</sub><sub>Δ</sub><sub>Nef (negative) and NL4.3-Nef</sub>
</p>
<p>SF2(positive) controls are
</p>
<p>shown.PanelB: Scatterplots depicting the ability of recombinant EC (red) and CP (blue) Nef recombinant viruses to down-regulate HLA-I are
shown.PanelC: Flow cytometry plots depicting representative staining of cell-surface CD74 (y-axis) vs. intracellular p24Gag<sub>(x-axis) using</sub>
</p>
<p>uninfected and control viruses are shown.PanelD: Scatterplots depicting the ability of recombinant EC (red) and CP (blue) Nef recombinant
viruses to upregulate CD74 are shown.PanelE: Flow cytometry plots depicting representative staining of cell-surface CD4 (y-axis) vs. GFP (x-axis)
after delivery of no-DNA,ΔNef and NefSF2plasmid vectors are shown.PanelF: Scatterplots depicting the ability of EC (red) and CP (blue)-derived
</p>
<p>Nef to downregulate CD4 are shown. In these experiments, results are normalized to NL4.3-NefSF2.(positive) and NL4.3-ΔNef (negative) controls.
</p>
<p>The activity of NL4.3-ΔNef orΔNef plasmid is inherently set to zero. Bars represent median and interquartile ranges. P-values were calculated
using Mann–Whitney U-test.
</p>
</div>
<div data-page='6' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 6</p><p>escape mutations that are unique to EC [26,27]. There-
fore, we used our EC dataset to identify HLA-associated
polymorphisms specific to this population, in a novel
analysis of this type. It is important to note that this ana-
lysis was aimed at identifying HLA-B*57-associated poly-
morphisms in the EC dataset itself (and is therefore
distinct from the comparative analyses described in [21]
that sought to identify polymorphisms enriched among
EC compared to CP). Statistical power is a major limita-
tion of this analysis, but HLA-associated polymorphisms
can be identified in modestly sized datasets (e.g. [37]),
</p>
<p>especially if analyses are limited to specific alleles. At
p &lt; 0.05, we identified nine B*57-associated polymorphic
sites specific to EC (Figure 6A), most of which differed
from B*57-associated polymorphisms commonly identi-
fied in population-level analyses [20,29]. In contrast,
B*57-associated polymorphisms identified in our CP data-
set using the same approach were largely as expected
[20,29,30]. ELISpot reactivity to overlapping peptides
spanning codons 28, 55, 85, 178 and 198 has been docu-
mented in B*57+ EC [27]; and V85L has been described
to function as an escape mutation in the B*57/58-KF9
</p>
<p><b>B*57</b> <b>Non B*57</b>
<b>0</b>
</p>
<p><b>50</b>
<b>100</b>
<b>150</b>
</p>
<p><b>B*57</b> <b>Non B*57</b>
<b>0</b>
</p>
<p><b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
</p>
<p><b>B*57</b> <b>Non B*57</b>
<b>0</b>
</p>
<p><b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
<b>120</b>
</p>
<p><b>B*57</b> <b>Non B*57</b>
<b>0</b>
</p>
<p><b>50</b>
<b>100</b>
<b>150</b>
<b>200</b>
<b>250</b>
</p>
<p><b>B*57</b> <b>Non B*57</b>
<b>0</b>
</p>
<p><b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
<b>120</b>
</p>
<p><b>Infectivity</b> <b>Viral replication</b>
</p>
<p><b>HLA-I down-regulation</b> <b>CD74 up-regulation</b>
</p>
<p><b>CD4 down-regulation</b>
</p>
<p><b>A</b> <b>B</b>
</p>
<p><b>C</b> <b>D</b>
</p>
<p><b>E</b>
</p>
<p><b>R</b>
</p>
<p><b>e</b>
</p>
<p><b>la</b>
</p>
<p><b>ti</b>
</p>
<p><b>v</b>
</p>
<p><b>e</b>
</p>
<p><b> a</b>
</p>
<p><b>c</b>
</p>
<p><b>ti</b>
</p>
<p><b>v</b>
</p>
<p><b>ity</b>
</p>
<p><b> to</b>
</p>
<p><b> c</b>
</p>
<p><b>o</b>
</p>
<p><b>n</b>
</p>
<p><b>tr</b>
</p>
<p><b>o</b>
</p>
<p><b>l (%</b>
</p>
<p><b>)</b>
</p>
<p><b>p= 0.7</b> <b>p= 0.2</b>
</p>
<p><b>p= 0.7</b> <b>p= 0.3</b>
</p>
<p><b>p= 0.038</b>
</p>
<p>Figure 5Relationship between host HLA-B*57 expression and Nef activities in EC.Scatterplots are shown depicting the ability of EC-
derived Nefs from B*57-expressing (closed red circles) and non-B*57 expressing (open red circles) to enhance virion infectivity (panelA), enhance
viral replication in PBMC (panelB; data depict means for all four PBMC donors), down-regulate HLA-I (panelC), up-regulate CD74 (panelD) and
down-regulate CD4 (panelE). All results are normalized to NefSF2. Bars represent median and interquartile ranges. P-values were calculated using
</p>
<p>Mann–Whitney U-test.
</p>
</div>
<div data-page='7' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 7</p><p><b>6</b> <b>7</b> <b>8</b> <b>9</b> <b>10</b>
<b>0</b>
</p>
<p><b>50</b>
<b>100</b>
</p>
<p><b>150</b> <b>Spearman R= 0.23</b>
</p>
<p><b>p= 0.4</b>
</p>
<p><b>6</b> <b>7</b> <b>8</b> <b>9</b> <b>10</b>
</p>
<p><b>0</b>
<b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
</p>
<p><b>100</b> <b>Spearman R= -0.50</b>
</p>
<p><b>p= 0.04</b>
</p>
<p><b>6</b> <b>7</b> <b>8</b> <b>9</b> <b>10</b>
</p>
<p><b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
</p>
<p><b>120</b> <b>Spearman R= -0.57</b>
</p>
<p><b>p= 0.02</b>
</p>
<p><b>6</b> <b>7</b> <b>8</b> <b>9</b> <b>10</b>
</p>
<p><b>0</b>
<b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
</p>
<p><b>120</b> <b>Spearman R= -0.60</b>
</p>
<p><b>p= 0.01</b>
</p>
<p><b>6</b> <b>7</b> <b>8</b> <b>9</b> <b>10</b>
</p>
<p><b>0</b>
<b>20</b>
<b>40</b>
<b>60</b>
<b>80</b>
<b>100</b>
</p>
<p><b>120</b> <b>Spearman R= -0.37</b>
</p>
<p><b>p= 0.1</b>
<b>Infectivity</b>
</p>
<p><b>Viral replication</b> <b>HLA-I down-regulation</b>
</p>
<p><b>CD74 up-regulation</b> <b>CD4 down-regulation</b>
</p>
<p><b>A</b>
</p>
<p><b>C</b>
</p>
<p><b>D</b> <b>E</b>
</p>
<p><b>G</b>
<b>F</b>
</p>
<p><b>Relative activity to control (%)</b>
</p>
<p>G R D
E
</p>
<p>Non-
adapted
</p>
<p>Adapted
</p>
<p>C V I
</p>
<p>L L
</p>
<p>M
G
Q
</p>
<p><b>B</b>
</p>
<p><b>Relative activity to control (%)</b>
</p>
<p><b># EC-specific B*57-assoc. polymorphisms</b>
</p>
<p>Figure 6HLA-B*57-associated polymorphisms in EC, and their relationship with Nef function.PanelA: Two-dimensional map of B*57-
associated polymorphisms identified in an exploratory analysis of the EC cohort using phylogenetically-corrected approaches with p&lt;0.05 [20].
“Nonadapted”forms (those less likely to be observed in the presence of B*57 at a specific location) are shown above the Nef protein in blue;“adapted”
forms (those enriched among B*57-expressing persons) are shown below the Nef protein in red. Due to limited statistical power and other reasons,
both nonadapted and adapted forms are not always identified at a given position. Boxed codons indicate those where the amino acid varied in B*57
EC; the remainder were either expressed in 100% of B*57 EC (adapted forms 3G and 19F) or 0% of B*57 EC (nonadapted forms 105Q, 178G, 198M). In
subsequent analyses, Nef sequences from B*57-expressing persons were counted as harboring a B*57-associated polymorphism at a given site if that
site expressed anything other than the nonadapted form, or the adapted form if no nonadapted form was identified.PanelB: The locations of the
nine EC-specific B*57-associated polymorphisms are indicated in green on the structure model of the Nef protein (composite crystal structure kindly
provided by Art F. Y. Poon, [31])PanelsC-G: Correlations between the number of B*57-associated polymorphisms in Nef sequences from B*57-
expressing EC, and five Nef functions evaluated. Statistical analyses were done using Spearman’s correlation.
</p>
</div>
<div data-page='8' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 8</p><p>epitope [38], further suggesting that these polymorphism
may be due to CTL selection pressure in B*57 EC. In
addition, paired Nef sequences from PBMC and plasma
were available for two B*57 EC in the present study. The
first exhibited identical amino acids in both compart-
ments at all 7 of the residues putatively associated with
HLA-B*57, while the second exhibited different amino
acids at three of the seven B*57-associated sites (codons
85, 105 and 198). Although it is not possible to make con-
clusions based on only two patients, these data support
evolution of these sitesin vivo.
</p>
<p>Notably, we observed dramatic inverse associations
between the number of EC-specific B*57-associated
polymorphisms and Nef-mediated replication, HLA-I
down-regulation and CD74 up-regulation. CD4 down-
regulation, the only function in which host expression of
B*57 was in itself significantly associated with poorer
function in EC, also displayed a modest, albeit not
</p>
<p>significant, negative relationship between the burden
of B*57-associated escape mutations and function
(Figure 6). These remarkable inverse relationships were
particular to B*57 and not observed for other HLA alleles
with similar frequency in our EC cohort. Taken together,
results suggest that HLA-B*57-associated CTL pressures
select for non-canonical polymorphisms in EC, which
contribute additively to multiple functional impairments
in EC Nef. Building upon previous studies of recombinant
viruses encodinggagandpolsequences from the same EC
cohort [3,4], our results support a complex relationship
between B*57-associated immune pressures and Nef
function.
</p>
<p>In an exploratory analysis of Nef amino acid sequences,
we identified 23 polymorphisms, located at 14 residues,
associated with Nef function in EC. None overlapped with
mutations previously identified in HIV non-progressors
[39] or with sites reported to affect HLA-I down-regulation
</p>
<p>Table 1 Analysis of Nef residues associated with EC Nef functions (all N&gt;5 and q&lt;0.4)
</p>
<p>Nef activity Position
</p>
<p>on HXB2
</p>
<p>AAa No. of subjectsb Relative Nef activity (%) p-
value
</p>
<p>q-
value
</p>
<p>With AA Without AA With AA Without AA
</p>
<p>HLA-I down-regulation 8 S 17 27 83.6 97.2 0.01 0.3
</p>
<p>8 R 14 30 106.1 91.1 0.02 0.3
</p>
<p>11 V 20 21 87.2 98.9 0.01 0.3
</p>
<p>14 P 34 11 91.1 108 0.004 0.3
</p>
<p>46 S 36 8 96.7 77.1 0.01 0.3
</p>
<p>108 D 34 11 91.1 105.8 0.02 0.3
</p>
<p>108 E 11 34 105.8 91.1 0.02 0.3
</p>
<p>138 C 5 40 79.8 96.5 0.02 0.3
</p>
<p>138 T 40 5 96.5 79.8 0.02 0.3
</p>
<p>192 H 36 9 97 78.1 0.01 0.3
</p>
<p>192 R 6 39 84 96.8 0.04 0.4
</p>
<p>205 N 22 23 101 83.6 0.02 0.3
</p>
<p>205 D 22 23 85.9 98.9 0.03 0.3
</p>
<p>CD74 up-regulation 8 S 17 27 37.7 60 0.004 0.3
</p>
<p>28 E 14 31 73.7 42.4 0.008 0.3
</p>
<p>28 D 29 16 40.4 65.3 0.009 0.3
</p>
<p>135 F 10 35 35 57.5 0.003 0.3
</p>
<p>135 Y 34 11 57.1 35.1 0.01 0.3
</p>
<p>CD4 down-regulation 8 S 17 27 84.8 92.6 0.007 0.3
</p>
<p>14 P 34 11 86.1 97.9 0.003 0.1
</p>
<p>14 S 5 40 102.9 87.2 0.02 0.4
</p>
<p>15 A 11 34 76.9 91.3 0.02 0.4
</p>
<p>21 K 11 34 70.7 91.6 0.002 0.1
</p>
<p>21 R 28 17 91.6 76.9 0.02 0.4
</p>
<p>105 R 12 33 76.6 91.9 0.003 0.1
</p>
<p>163 C 9 36 98.1 86.6 0.02 0.4
</p>
<p>a
</p>
<p>AA, amino acid.
</p>
<p>b
</p>
<p>Row totals vary depending on the codon position because sequences with gaps in the alignment are considered missing data.
</p>
</div>
<div data-page='9' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 9</p><p>activity in chronic infection [40], a discrepancy that might
be due to non-canonical polymorphisms observed in EC.
Of interest, 8S was associated with Nef-mediated modula-
tion of cell-surface CD4, HLA-I and CD74, an observation
consistent with codon 8’s involvement in myristylation [41].
Also, 28D and 105X (in this case R), associated with modu-
lation of CD74 and CD4, respectively, are EC-specific B*57-
associated polymorphisms identified in the present study.
</p>
<p>Viral genetic studies of EC feature numerous challenges
and limitations. Although care was taken to choose a Nef
clone that reflected each patient’s original bulk HIV RNA
sequence, and to rule out proviral DNA contamination, po-
tential biases associated with PCR amplification from
extremely low copy-number templates must be acknowl-
edged. On the other hand, use of a single sequence per pa-
tient alleviates potential biases associated with quasispecies
approaches to compare samples with low vs. high genetic
diversity. As our goal was to specifically investigate the
function of Nef in EC, we employed recombinant virus
(and single-protein expression) approaches to eliminate
potential confounding effects of other HIV-1 proteins; how-
ever, such approaches may not reflect the characteristics of
infectious molecular clones or whole-virus isolates recov-
ered from PBMC, procedures that are rarely successful in
EC [21,42]. Recombinant virus approaches are also inher-
ently limited by potential incompatibilities between insert
and backbone; our choice of a recombinant control strain
(NL4.3-NefSF2) alleviates this to a minor extent. Although
</p>
<p>we assessed Nef activity using primary PBMCs and immor-
talized cell lines, Nef’s multiple functions [16-18,43] may
vary in different cell types [43-45]. Furthermore, thein vivo
relevance of our observations - in particular, the extent to
which these functional differences contribute to the viremia
control in EC - remains unclear. Although our results are
consistent with the transmission of partially attenuated Nef
sequences in at least some EC, and/or further immune-
mediated attenuation in others [3,4,46], it is not possible to
disentangle cause and effect in cross-sectional studies. Fur-
thermore, Nef function may change over the infection
course [47], therefore longitudinal analysis of Nef function
in controllers, beginning in the acute/early phase of infec-
tion, is warranted. Finally, although our results suggest that
non-canonical polymorphisms in EC may contribute to
attenuated Nef function, it will be important to validate
these findings in larger EC cohorts. Despite these limita-
tions, our study represents the largest linked analysis of
multiple in vitro Nef functions in EC to date, and to our
knowledge the only study assessing CD74 up-regulation
and replication capacity in this group.
</p>
<p>Conclusions
</p>
<p>EC Nef clones were generally functional; however, all
five activities assessed were significantly impaired com-
pared to CP Nef clones. HLA-I-restricted immune
</p>
<p>pressure, most notably by B*57, may contribute to the
differences observed. Taken together with previous stud-
ies of HIV Gag, Pol, and Env function in EC [3,4,7], our
results support decreased viral protein function as a hall-
mark of the EC phenotype and underscore the potential
role of immune pressures in modulating viral protein
function in this rare group.
</p>
<p>Methods
Study subjects
</p>
<p>45 EC (median [interquartile range, IQR] pVL 2 [0.2-14]
RNA copies/ml [25]; median [IQR] CD4 count 811
[612–1022] cells/mm3
</p>
<p>) and 46 CP (median [IQR] pVL
80500 [25121–221250] RNA copies/ml); median [IQR]
CD4 count 292.5 [72.5-440] cells/mm3) were studied as
described previously [3,4,21,25]. All EC and CP were
HIV-1 subtype B-infected, untreated at the time of sam-
ple collection, recruited from the Boston area, and com-
parable with respect to ethnicity and date of HIV
diagnosis (1985–2006 for EC vs. 1981–2003 for CP).
This study was approved by the institutional review
board of Massachusetts General Hospital, Boston USA;
all participants provided written informed consent.
</p>
<p>Cloning and analysis of nef genes
</p>
<p>For EC, HIV RNA was extracted from a starting volume
of 4.5 to 35.0 ml of plasma and amplified using nested
RT-PCR, as described [21]. Given that the median pVL
in our EC cohort was 2 RNA copies/ml [IQR 0.2-14]
[25], we estimate that on average, 40 viral RNA tem-
plates were extracted for each EC patient. To rule out
proviral DNA contamination, all extractions included a
DNAse treatment step; controls lacking RT enzyme were
also performed [21]. For CP, HIV RNA was extracted
from 0.5ml of plasma and amplified in the same manner.
Nef amplicons were cloned into pIRES2-EGFP expres-
sion vector (Clontech). A minimum of three Nef clones
were sequenced per patient, and a single clone with an
intact Nef reading frame that closely resembled the ori-
ginal bulk plasma RNA sequence was selected. Genbank
accession numbers for clonal Nef sequences are
JX171199-JX171243 (EC) and JX440926-JX440971 (CP).
</p>
<p>Recombinant virus construction and verification of Nef
expression
</p>
<p>Nefclones were transferred into a pNL4.3ΔNef plasmid
as described [23] and confirmed by DNA sequencing.
Recombinant viruses harboring neffrom HIV strain SF2
(NL4.3-NefSF2), and lacking nef (NL4.3ΔNef ) were used
</p>
<p>as positive and negative controls, respectively. Infectious
viruses were generated as described [48]. Briefly, HEK-
293T cells were transfected with each proviral clone.
Virus-containing culture supernatants were harvested
48 hr following transfection, titered by p24Gag ELISA
</p>
</div>
<div data-page='10' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 10</p><p>(ZeptoMetrix Corp.) and aliquots stored at −80°C
until use.
</p>
<p>HEK-293T cell pellets collected at this time point were
used to prepare total cell lysates as described [45] that were
subjected to SDS-PAGE in duplicate and transferred to
nitrocellulose membranes. Nef genetic diversity poses a
challenge to antibody-based detection as differences in
reactivity may reflect suboptimal antibody binding rather
than variation in protein levels. To ensure detection of
patient-derived Nef, duplicate blots were probed using
unique anti-Nef polyclonal antisera, developed from sheep
(ARP 444; provided by O.T. Fackler, Heidelberg University,
Germany) or rabbit (NIH AIDS Research and Reference
Reagent Program). Band intensities were quantified using
ImageQuant LAS 4000 (GE Healthcare Life Sciences).
</p>
<p>Virion infectivity and replication assays
</p>
<p>Recombinant virus infectivity was determined by expos-
ing 104TZM-bl cells (NIH AIDS Research and Reference
Reagent Program) to 3 ng p24Gag recombinant virus
followed by chemiluminescence detection 48 hr later as
described [49]. Infectivity values represented the mean
of triplicate experiments, normalized to control strain
NL4.3-NefSF2, such that values &gt; 100% and &lt; 100% indi-
</p>
<p>cated increased or decreased infectivity, respectively.
Recombinant virus replication was assessed by exposing
106freshly isolated PBMC from four HIV-negative donors
to 10 ng p24Gagrecombinant virus for 8 hr, washing twice,
and then resuspending cells in a culture medium (RPMI
1640, 10% FCS) as described [22,23]. Three days later,
PBMCs were stimulated with phytohemagglutinin at
5μg/ml. Culture supernatants were collected and replaced
with fresh medium supplemented with human rIL-2 every
3 days. Viral replication was monitored by measuring
p24Gag in the culture supernatant using ELISA over 12
days. ELISA values during the initial burst of viral replica-
tion (on day 9) were used as our measure of replication
capacity. Results were expressed as the mean of quadrupli-
cate assessments in each donor, normalized to control
strain NL4.3-NefSF2.
</p>
<p>Analysis of receptor modulation HIV-infected cell surface
To assess Nef-mediated HLA-I down-regulation and
CD74 up-regulation, 721.221 cells stably expressing CD4
and HLA-A*24:02 (provided by Masafumi Takiguchi,
Kumamoto University, Japan) were exposed to 300 ng
p24Gagrecombinant virus for 48 hr, followed by staining
with anti-HLA-A24-PE (MBL), anti-CD74-Alexa Fluor-
647 (BioLegend), 7-amino-actinomycin D (BioLegend),
and anti-p24 Gag-FITC (Beckman Coulter), as described
[23]. Mean fluorescence intensity (MFI) of each receptor
in live p24Gag positive and negative subsets was deter-
mined by flow cytometry (FACS Canto II; BD Bios-
ciences). Results were expressed as the mean of
</p>
<p>duplicate experiments, normalized to control strain
NL4.3-NefSF2.
</p>
<p>Analysis of Nef-mediated CD4 down-regulation
</p>
<p>To assess Nef-mediated CD4 down-regulation, 3 × 105
CEM-SS cells were transfected with 5 µg plasmid DNA
encoding Nef protein and GFP by electroporation (BioRad
GenePulser MX) and stained 24 hr later with anti-CD4-
APC (BD Biosciences). MFI of GFP-negative and GFP-
positive (Nef-expressing) subsets was determined by flow
cytometry (Guava easyCyte 8HT, Millipore). Results were
normalized to plasmid expressing NefSF2.
</p>
<p>Statistical analyses, including identification of B*57-
associated polymorphisms in patient-derived Nef
sequences
</p>
<p>Phylogenetically-informed methods were used to identify
Nef amino acids significantly associated with HLA-
B*57allele expression in our EC ([28], implemented at
http://research.microsoft.com/en-us/um/redmond/projects/
mscompbio/phylododdsratio/default.aspx). Multiplecom-
parisons were addressed using q-values, the p-value
analogue of the false discovery rate (FDR) [50]. The FDR
is the expected proportion of false positives among
results deemed significant at a given p-value threshold;
for example, at a q ≤ 0.2, we expect 20% of identified
associations to be false positives.
</p>
<p>Additional file
</p>
<p>Additional file 1:Table S1.EC Nef activity with aligned amino acid
sequence.
</p>
<p>Competing interests
</p>
<p>The authors declare that they have no competing interests.
</p>
<p>Authors’contributions
</p>
<p>PM, TJM, ZLB, MAB, and TU designed the study; PM, TJM, EM, YO, XTK, MT,
and MM performed the experiments; FP, TM and BDW provided access to
patient samples and analyzed clinical data; PM, TJM, EM, ZLB, MAB, and TU
analyzed data; and PM, ZLB, MAB, and TU wrote the paper. All authors read
and approved the final manuscript.
</p>
<p>Acknowledgments
</p>
<p>We thank Dr. Jonathan Carlson for helpful discussions. This study was
supported by a grant-in-aid for scientific research and a Global COE Program
(Global Education and Research Center Aiming at the control of AIDS) from
the Ministry of Education, Science, Sports, and Culture of Japan, a grant-in-
aid for AIDS research from the Ministry of Health, Labor, and Welfare of
Japan (to TU), and the Canadian Institutes for Health Research (CIHR)
operating grant MOP-93536 (to ZLB and MAB). We acknowledge and thank
the International HIV Controllers study, funded by Bill and Melinda Gates
Foundation, the Schwartz Foundation and the Harvard University Center for
AIDS Research. PM is a postdoctoral fellow who received support from the
Japan AIDS Foundation and the Global Health Research Initiative (GHRI), a
collaborative research funding partnership of the CIHR, the Canadian
International Development Agency, and the International Development
Research Centre. EM is supported by a Master’s Scholarship from the
Canadian Association of HIV Research and Abbott Virology. ZLB is the
recipient of a CIHR New Investigator Award and a Scholar Award from the
</p>
</div>
<div data-page='11' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 11</p><p>Michael Smith Foundation for Health Research. MAB holds a Canada
Research Chair, Tier 2, in Viral Pathogenesis and Immunity.
</p>
<p>Author details
</p>
<p>1<sub>Center for AIDS Research, Kumamoto University, 2-2-1 Honjo, Kumamoto</sub>
</p>
<p>860-0811, Japan.2Simon Fraser University, Burnaby, BC, Canada.3Ragon
Institute of Massachusetts General Hospital, Massachusetts Institute of
Technology and Harvard University, Boston, MA, USA.4University of Tokyo,
Tokyo, Japan.5<sub>British Columbia Centre for Excellence in HIV/AIDS, Vancouver,</sub>
</p>
<p>BC, Canada.
</p>
<p>Received: 28 November 2012 Accepted: 21 December 2012
Published: 7 January 2013
</p>
<p>References
</p>
<p>1. Pereyra F, Jia X, McLaren PJ, Telenti A, de Bakker PI, Walker BD, Ripke S,
Brumme CJ, Pulit SL, Carrington M,et al:The major genetic determinants
of HIV-1 control affect HLA class I peptide presentation.Science2010,
330:1551–1557.
</p>
<p>2. Chen H, Ndhlovu ZM, Liu D, Porter LC, Fang JW, Darko S, Brockman MA,
Miura T, Brumme ZL, Schneidewind A,et al:TCR clonotypes modulate the
protective effect of HLA class I molecules in HIV-1 infection.Nat Immunol
2012,13:691<sub>–</sub>700.
</p>
<p>3. Brumme ZL, Li C, Miura T, Sela J, Rosato PC, Brumme CJ, Markle TJ, Martin E,
Block BL, Trocha A,et al:Reduced replication capacity of NL4-3
recombinant viruses encoding reverse transcriptase-integrase sequences
from HIV-1 elite controllers.J Acquir Immune Defic Syndr2011,56:100–108.
4. Miura T, Brockman MA, Brumme ZL, Brumme CJ, Pereyra F, Trocha A, Block
</p>
<p>BL, Schneidewind A, Allen TM, Heckerman D, Walker BD:HLA-associated
alterations in replication capacity of chimeric NL4-3 viruses carrying gag-
protease from elite controllers of human immunodeficiency virus type 1.
J Virol2009,83:140–149.
</p>
<p>5. Deacon NJ, Tsykin A, Solomon A, Smith K, Ludford-Menting M, Hooker DJ,
McPhee DA, Greenway AL, Ellett A, Chatfield C,et al:Genomic structure of
an attenuated quasi species of HIV-1 from a blood transfusion donor
and recipients.Science1995,270:988<sub>–</sub>991.
</p>
<p>6. Kirchhoff F, Greenough TC, Brettler DB, Sullivan JL, Desrosiers RC:Brief
report: absence of intact nef sequences in a long-term survivor with
nonprogressive HIV-1 infection.N Engl J Med1995,332:228–232.
7. Lassen KG, Lobritz MA, Bailey JR, Johnston S, Nguyen S, Lee B, Chou T,
</p>
<p>Siliciano RF, Markowitz M, Arts EJ:Elite Suppressor–Derived HIV-1
Envelope Glycoproteins Exhibit Reduced Entry Efficiency and Kinetics.
PLoS Pathog2009,5:e1000377.
</p>
<p>8. Kestler HW III, Ringler DJ, Mori K, Panicali DL, Sehgal PK, Daniel MD,
Desrosiers RC:Importance of the nef gene for maintenance of high virus
loads and for development of AIDS.Cell1991,65:651–662.
</p>
<p>9. Daniel MD, Kirchhoff F, Czajak SC, Sehgal PK, Desrosiers RC:Protective
effects of a live attenuated SIV vaccine with a deletion in the nef gene.
Science1992,258:1938<sub>–</sub>1941.
</p>
<p>10. Zaunders JJ, Geczy AF, Dyer WB, McIntyre LB, Cooley MA, Ashton LJ,
Raynes-Greenow CH, Learmont J, Cooper DA, Sullivan JS:Effect of long-
term infection with nef-defective attenuated HIV type 1 on CD4+ and
CD8+ T lymphocytes: increased CD45RO+CD4+ T lymphocytes and
limited activation of CD8+ T lymphocytes.AIDS Res Hum Retroviruses1999,
15:1519–1527.
</p>
<p>11. Garcia JV, Miller AD:Serine phosphorylation-independent downregulation
of cell-surface CD4 by nef.Nature1991,350:508<sub>–</sub>511.
</p>
<p>12. Schwartz O, Marechal V, Le Gall S, Lemonnier F, Heard JM:Endocytosis of
major histocompatibility complex class I molecules is induced by the
HIV-1 Nef protein.Nat Med1996,2:338<sub>–</sub>342.
</p>
<p>13. Schindler M, Wurfl S, Benaroch P, Greenough TC, Daniels R, Easterbrook P,
Brenner M, Munch J, Kirchhoff F:Down-modulation of mature major
histocompatibility complex class II and up-regulation of invariant chain
cell surface expression are well-conserved functions of human and
simian immunodeficiency virus nef alleles.J Virol2003,77:10548–10556.
14. Münch J, Rajan D, Schindler M, Specht A, Rücker E, Novembre FJ, Nerrienet
</p>
<p>E, Müller-Trutwin MC, Peeters M, Hahn BH, Kirchhoff F:Nef-Mediated
Enhancement of Virion Infectivity and Stimulation of Viral Replication
Are Fundamental Properties of Primate Lentiviruses.J Virol2007,
81:13852–13864.
</p>
<p>15. Miller MD, Warmerdam MT, Gaston I, Greene WC, Feinberg MB:The human
immunodeficiency virus-1 nef gene product: a positive factor for viral
infection and replication in primary lymphocytes and macrophages.
J Exp Med1994,179:101–113.
</p>
<p>16. Arhel NJ, Kirchhoff F:Implications of Nef: host cell interactions in viral
persistence and progression to AIDS.Curr Top Microbiol Immunol2009,
339:147<sub>–</sub>175.
</p>
<p>17. Arien KK, Verhasselt B:HIV Nef: role in pathogenesis and viral fitness.
Curr HIV Res2008,6:200–208.
</p>
<p>18. Foster JL, Denial SJ, Temple BR, Garcia JV:Mechanisms of HIV-1 Nef
function and intracellular signaling.J Neuroimmune Pharmacol2011,
6:230<sub>–</sub>246.
</p>
<p>19. Fackler OT, Alcover A, Schwartz O:Modulation of the immunological
synapse: a key to HIV-1 pathogenesis?Nat Rev Immunol2007,7:310–317.
20. Carlson JM, Brumme CJ, Martin E, Listgarten J, Brockman MA, Le AQ, Chui C,
</p>
<p>Cotton LA, Knapp DJ, Riddler SA,et al:Correlates of protective cellular
immunity revealed by analysis of population-level immune escape
pathways in HIV-1.J Virol2012,86:13202–13216.
</p>
<p>21. Miura T, Brockman MA, Brumme CJ, Brumme ZL, Carlson JM, Pereyra F,
Trocha A, Addo MM, Block BL, Rothchild AC,et al:Genetic characterization
of human immunodeficiency virus type 1 in elite controllers: lack of
gross genetic defects or common amino acid changes.J Virol2008,
82:8422–8430.
</p>
<p>22. Fackler OT, Moris A, Tibroni N, Giese SI, Glass B, Schwartz O, Krausslich HG:
Functional characterization of HIV-1 Nef mutants in the context of viral
infection.Virology2006,351:322–339.
</p>
<p>23. Ueno T, Motozono C, Dohki S, Mwimanzi P, Rauch S, Fackler OT, Oka S,
Takiguchi M:CTL-mediated selective pressure influences dynamic
evolution and pathogenic functions of HIV-1 Nef.J Immunol2008,
180:1107<sub>–</sub>1116.
</p>
<p>24. Neri F, Giolo G, Potesta M, Petrini S, Doria M:CD4 downregulation by the
human immunodeficiency virus type 1 Nef protein is dispensable for
optimal output and functionality of viral particles in primary T cells.
J Gen Virol2011,92:141–150.
</p>
<p>25. Pereyra F, Addo MM, Kaufmann DE, Liu Y, Miura T, Rathod A, Baker B,
Trocha A, Rosenberg R, Mackey E,et al:Genetic and immunologic
heterogeneity among persons who control HIV infection in the absence
of therapy.J Infect Dis2008,197:563–571.
</p>
<p>26. Miura T, Brockman MA, Schneidewind A, Lobritz M, Pereyra F, Rathod A,
Block BL, Brumme ZL, Brumme CJ, Baker B,et al:HLA-B57/B*5801 human
immunodeficiency virus type 1 elite controllers select for rare gag
variants associated with reduced viral replication capacity and strong
cytotoxic T-lymphocyte [corrected] recognition.J Virol2009,
83:2743<sub>–</sub>2755.
</p>
<p>27. Bailey JR, Brennan TP, O'Connell KA, Siliciano RF, Blankson JN:Evidence of
CD8+ T-Cell-Mediated Selective Pressure on Human Immunodeficiency
Virus Type 1 nef in HLA-B*57+ Elite Suppressors.J Virol2009,83:88<sub>–</sub>97.
28. Carlson JM, Listgarten J, Pfeifer N, Tan V, Kadie C, Walker BD, Ndung'u T,
</p>
<p>Shapiro R, Frater J, Brumme ZL,et al:Widespread impact of HLA
restriction on immune control and escape pathways of HIV-1.J Virol
2012,86:5230–5243.
</p>
<p>29. Brumme ZL, John M, Carlson JM, Brumme CJ, Chan D, Brockman MA,
Swenson LC, Tao I, Szeto S, Rosato P,et al:HLA-associated immune escape
pathways in HIV-1 subtype B Gag, Pol and Nef proteins.PLoS One
2009,4:e6687.
</p>
<p>30. Frater AJ, Brown H, Oxenius A, Gunthard HF, Hirschel B, Robinson N, Leslie
AJ, Payne R, Crawford H, Prendergast A,et al:Effective T-cell responses
select human immunodeficiency virus mutants and slow disease
progression.J Virol2007,81:6742–6751.
</p>
<p>31. Poon AF, Swenson LC, Dong WW, Deng W, Kosakovsky Pond SL, Brumme
ZL, Mullins JI, Richman DD, Harrigan PR, Frost SD:Phylogenetic analysis of
population-based and deep sequencing data to identify coevolving sites
in the nef gene of HIV-1.Mol Biol Evol2010,27:819–832.
</p>
<p>32. Zuo J, Suen J, Wong A, Lewis M, Ayub A, Belzer M, Church J, Yang OO,
Krogstad P:Functional analysis of HIV type 1 Nef gene variants from
adolescent and adult survivors of perinatal infection.AIDS Res Hum
Retroviruses2012,28:486–492.
</p>
<p>33. Nou E, Zhou Y, Nou DD, Blankson JN:Effective Downregulation of HLA-
A*2 and HLA-B*57 by Primary Human Immunodeficiency Virus Type 1
Isolates Cultured from Elite Suppressors.J Virol2009,83:6941<sub>–</sub>6946.
</p>
</div>
<div data-page='12' style='padding: 20px;margin: 10px auto;max-width: 800px;'><p>Page 12</p><p>34. Casartelli N, Di Matteo G, Potestà M, Rossi P, Doria M:CD4 and Major
Histocompatibility Complex Class I Downregulation by the Human
Immunodeficiency Virus Type 1 Nef Protein in Pediatric AIDS
Progression.J Virol2003,77:11536–11545.
</p>
<p>35. Tobiume M, Takahoko M, Yamada T, Tatsumi M, Iwamoto A, Matsuda M:
Inefficient enhancement of viral infectivity and CD4 downregulation by
human immunodeficiency virus type 1 Nef from Japanese long-term
nonprogressors.J Virol2002,76:5959–5965.
</p>
<p>36. Corro G, Rocco C, De Candia C, Catano G, Turk G, Aulicino P, Bologna R, Sen
L:Genetic and functional analysis of HIV-1 nef gene derived from LTNP
children: association of attenuated variants with slow progression to
pediatric AIDS.AIDS Res Hum Retroviruses2012,28:1617–1626.
</p>
<p>37. Bhattacharya T, Daniels M, Heckerman D, Foley B, Frahm N, Kadie C, Carlson
J, Yusim K, McMahon B, Gaschen B,et al:Founder effects in the
assessment of HIV polymorphisms and HLA allele associations.Science
2007,315:1583–1586.
</p>
<p>38. O'Connell KA, Xu J, Durbin AP, Apuzzo LG, Imteyaz H, Williams TM, Ray SC,
Margolick JB, Siliciano RF, Blankson JN:HIV-1 evolution following
transmission to an HLA-B*5801-positive patient.J Infect Dis2009,
200:1820–1824.
</p>
<p>39. Kirchhoff F, Easterbrook PJ, Douglas N, Troop M, Greenough TC, Weber J,
Carl S, Sullivan JL, Daniels RS:Sequence variations in human
</p>
<p>immunodeficiency virus type 1 Nef are associated with different stages
of disease.J Virol1999,73:5497–5508.
</p>
<p>40. Lewis MJ, Lee P, Ng HL, Yang OO:Immune selection in vitro reveals
human immunodeficiency virus type 1 Nef sequence motifs important
for its immune evasion function in vivo.J Virol2012,86:7126–7135.
41. Geyer M, Fackler OT, Peterlin BM:Structure–function relationships in HIV-1
</p>
<p>Nef.EMBO Rep2001,2:580–585.
</p>
<p>42. Julg B, Pereyra F, Buzon MJ, Piechocka-Trocha A, Clark MJ, Baker BM, Lian J,
Miura T, Martinez-Picado J, Addo MM, Walker BD:Infrequent recovery of
HIV from but robust exogenous infection of activated CD4(+) T cells in
HIV elite controllers.Clin Infect Dis2010,51:233–238.
</p>
<p>43. Kirchhoff F, Schindler M, Specht A, Arhel N, Munch J:Role of Nef in
primate lentiviral immunopathogenesis.Cell Mol Life Sci2008,
65:2621–2636.
</p>
<p>44. Suzu S, Harada H, Matsumoto T, Okada S:HIV-1 Nef interferes with M-CSF
receptor signaling through Hck activation and inhibits M-CSF
bioactivities.Blood2005,105:3230–3237.
</p>
<p>45. Mwimanzi P, Hasan Z, Hassan R, Suzu S, Takiguchi M, Ueno T:Effects of
naturally-arising HIV Nef mutations on cytotoxic T lymphocyte
recognition and Nef's functionality in primary macrophages.
Retrovirology2011,8:50.
</p>
<p>46. Lobritz MA, Lassen KG, Arts EJ:HIV-1 replicative fitness in elite controllers.
Curr Opin HIV AIDS2011,6:214–220.
</p>
<p>47. Carl S, Greenough TC, Krumbiegel M, Greenberg M, Skowronski J, Sullivan
JL, Kirchhoff F:Modulation of different human immunodeficiency virus
type 1 Nef functions during progression to AIDS.J Virol2001,
75:3657–3665.
</p>
<p>48. Mwimanzi P, Hasan Z, Tokunaga M, Gatanaga H, Oka S, Ueno T:Naturally
arising HIV-1 Nef variants conferring escape from cytotoxic T
lymphocytes influence viral entry co-receptor expression and
susceptibility to superinfection.Biochem Biophys Res Commun2010,
403:422–427.
</p>
<p>49. Wei X, Decker JM, Liu H, Zhang Z, Arani RB, Kilby JM, Saag MS, Wu X, Shaw
GM, Kappes JC:Emergence of resistant human immunodeficiency virus
type 1 in patients receiving fusion inhibitor (T-20) monotherapy.
Antimicrob Agents Chemother2002,46:1896–1905.
</p>
<p>50. Storey JD, Tibshirani R:Statistical significance for genomewide studies.
Proc Natl Acad Sci U S A2003,100:9440–9445.
</p>
<p>doi:10.1186/1742-4690-10-1
</p>
<p>Cite this article as:Mwimanziet al.:Attenuation of multiple Nef
functions in HIV-1 elite controllers.Retrovirology201310:1.
</p>
<p><b>Submit your next manuscript to BioMed Central</b>
<b>and take full advantage of: </b>
</p>
<p><b>• Convenient online submission</b>
</p>
<p><b>• Thorough peer review</b>
</p>
<p><b>• No space constraints or color ﬁgure charges</b>
</p>
<p><b>• Immediate publication on acceptance</b>
</p>
<p><b>• Inclusion in PubMed, CAS, Scopus and Google Scholar</b>
</p>
<p><b>• Research which is freely available for redistribution</b>
</p>
<p>Submit your manuscript at 
www.biomedcentral.com/submit
</p>
</div></body>